sys/shell/Makefile.dep

```diff
@@ -216,6 +216,9 @@ endif
 ifneq (,$(filter shell_cmd_mci,$(USEMODULE)))
   USEMODULE += mci
 endif
+ifneq (,$(filter shell_cmd_md5sum,$(USEMODULE)))
+  USEMODULE += shell_cmd_vfs
```

Comment:
I'm afraid that's not enough.
You will also have to update the `#if MODULE_MD5SUM` in `sc_vfs.c`